docs/install-apps.md

```diff
@@ -10,7 +10,7 @@ The following are the parameters supported by this goal in addition to the [comm
 | --------  | ----------- | -------  |
 | appsDirectory | The server's `apps` or `dropins` directory where the application files should be copied. The default value is set to `apps` if the application is defined in the server configuration, otherwise it is set to `dropins`.  | No |
 | stripVersion | Strip artifact version when copying the application to Liberty runtime's application directory. The default value is `false`. | No |
-| installAppPackages | The Maven packages to copy to Liberty runtime's application directory. One of `dependencies`, `project` or `all`. The default is `dependencies`. | No |
+| installAppPackages | The Maven packages to copy to Liberty runtime's application directory. One of `dependencies`, `project` or `all`. The default is `dependencies`.<br>For ear type project, this parameter is ignored and only the project package will be copied. | No |
```

Comment:
Suggestion for that last sentence:

For an `ear` type project, this parameter is ignored and only the project package is installed.